ui/src/app/workflows/components/workflows-row/workflows-row.tsx

```diff
@@ -34,7 +34,7 @@ export class WorkflowsRow extends React.Component<WorkflowsRowProps, WorkflowRow
     public render() {
         const wf = this.props.workflow;
         return (
-            <div className='workflows-list__row-container'>
+            <div className='workflows-list__row-container workflows-cron-list__row-container'>
```

Comment:
mmmm this won't quite work either -- `WorkflowsRow` is used in multiple components, so they will _all_ have this class now.

You will have to select on a class that is _unique_ to the `CronWorkflowDetails` component

Reply:
> You will have to select on a class that is unique to the CronWorkflowDetails component.

![image](https://github.com/argoproj/argo-workflows/assets/100748721/170e48c8-b412-4d97-aff3-5484bf33dbeb)

```
.workflows-cron-list > .workflow-list {
    &__status {
        max-width: 80px;
        display: flex;
        align-items: center;

        /* checkboxes are not visible and are unused on this page */
        &--checkbox {
            appearance: none;
        }
    }
}
```

hello 😄

Let's give the class to cron-workflow-details and not to the workflows-row component!

I don't have a proper parent element at the moment, so I was thinking of adding a div element like the one in the image and then giving it a class and using the child class selector as advised, but what do you think?

Reply:
To not create an extra `div`, you could add the class on the same `div` as `argo-table-list`. You could use a [descendant combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/Descendant_combinator) instead of a direct child then

Reply:
Oh I missed the `argo-table-list` div, thanks I understand 👍